modules/assim.sequential/R/downscale_function.R

```diff
@@ -112,64 +153,85 @@ SDA_downscale_preprocess <- function(data_path, coords_path, date, carbon_pool)
 ##'
 ##' @return A list containing the training and testing data sets, models, predicted maps for each ensemble member, and predictions for testing data.
 
-SDA_downscale <- function(preprocessed, date, carbon_pool, covariates, model_type = "rf", seed = NULL) {
+SDA_downscale <- function(preprocessed, date = NULL, carbon_pool, covariates, model_type = "rf", seed = NULL) {
+  if (!missing(date)) {
+    ## If you see this and it is after 2026-04-01, please remove
+    #    1. line starting with ##' @param date
+    #    2. date = NULL from function call
+    #    3. this conditional starting with if (!missing(date)) and ending after the warning below
+    PEcAn.logger::logger.warn("'date' argument is not used and will be removed on or after 2026-04-01. It is currently ignored.", call. = FALSE)
```

Comment:
logger.warn doesn't take `call.` and treats its contents as another message to print
```suggestion
    PEcAn.logger::logger.warn("'date' argument is not used and will be removed on or after 2026-04-01. It is currently ignored.")
```

(Also `call.` is high up my list of the base R arguments that look the most like typos, but that's not our problem to fix 😉 )